ospfd/ospf_opaque.c

```diff
@@ -1621,10 +1630,13 @@ ospf_opaque_lsa_install (struct ospf_lsa *lsa, int rt_recalc)
 struct ospf_lsa *
 ospf_opaque_lsa_refresh (struct ospf_lsa *lsa)
 {
-  struct ospf *ospf;
+  struct ospf *ospf = NULL;
   struct ospf_opaque_functab *functab;
   struct ospf_lsa *new = NULL;
   
+  if (IS_DEBUG_OSPF_TRACE)
+    zlog_debug ("%s: default ospf lookup. opaque lsa->oi->ospf->vrf_id %u",
+                __PRETTY_FUNCTION__, lsa->oi->ospf->vrf_id);
   ospf = ospf_lookup ();
```

Comment:
Most places, I have addressed to use vrf aware apis. There are places still required scoping to make vrf aware, snmp, ospf ri etc. Currently primary functionality is working with VRF aware. I will address them time to come ...